specification/web/resource-manager/Microsoft.Web/stable/2018-02-01/ClassicMobileServices.json

```diff
@@ -0,0 +1,264 @@
+{
```

Comment:
@ruslany is this the first version of this Swagger? Or is there a previous API version?

Reply:
@ruslany We should not expose the ClassicMobileServices.json. It is not really useful for customers and was only used during transition (Zumo -> ARM web apps).